helm-app/templates/phpmyadmin.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.phpmyadmin.name }}
+  namespace: {{ .Values.phpmyadmin.namespace }}
+spec:
+  replicas: {{ .Values.phpmyadmin.replicas }}
+  selector:
+    matchLabels:
+      app: {{ .Values.phpmyadmin.name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.phpmyadmin.name }}

```

Comment:
Any particular reason I am not able to add any more labels?

Reply:
fixed.